dbt_project.yml

```diff
@@ -1058,8 +1058,13 @@ models:
       +schema: glacier
       avalanche_c:
         +schema: glacier_avalanche_c
+
+    lido_liquidity:
+      +schema: lido_liqudity
+      kyberswap_ethereum:
+       +schema: lido_liquidity_kyberswap_ethereum
```

Comment:
i'm a bit confused on what you'd like the spell schema / table name to be. in the current setup, it'll be:
`lido_liquidity_kyberswap_ethereum.lido_liquidity_kyberswap_ethereum`
based on schema value here, the spell living in that directory path & the alias value in the spell config block. in the post_hook to display in data explorer, there's also a `lido` value, which differs from `lido_liqudity` here

i'd recommend looking at it from end name desired and working backwards, to figure out the values to apply in spellbook. ultimately, you'll end up revising directory names, file names, values in project file here to match directory/file naming, then finally the alias field for table name.

if this makes sense, please do update, otherwise feel free to send ideal schema/table name output and we can help reflect that

Reply:
Hello, sorry for the delay, thank you very much for your comments, I changed naming according to suggestions. The only thing that remained unchanged is day/time naming. Give more context in the comment below that issue.